src/v/container/chunked_vector.h

```diff
@@ -108,22 +108,12 @@ class chunked_vector {
       : chunked_vector(elems.begin(), elems.end()) {}
 
     /**
-     * @brief Construct a new vector by moving from a given range
-     */
-    template<typename Range>
-    requires std::ranges::sized_range<Range>
-    explicit chunked_vector(Range range)
-      : chunked_vector() {
-        reserve(std::ranges::size(range));
-        std::move(range.begin(), range.end(), std::back_inserter(*this));
-    }
-
-    /**
-     * @brief Construct a new vector by copying from a const range
+     * @brief Construct a new vector from a range
```

Comment:
Perhaps it is work clarifying the semantics, e.g., I would look at this signature and expect that rvalue ranges (e.g., rvalue containers) would be moved from, but in fact it is only specific special ranges whose begin() and end() methods expose iterators which return rvalues, right?